Oxide.Core/Libraries/Lang.cs

```diff
@@ -204,6 +204,11 @@ private void AddLangFile(string file, Dictionary<string, string> langFile, Plugi
         {
             if (string.IsNullOrEmpty(plugin)) return null;
 
+            foreach (char invalidChar in Path.GetInvalidFileNameChars())
+            {
+                lang = lang.Replace(invalidChar, '_');
```

Comment:
Would probably be smart to actually shorten lang codes to two digits as well, as that's generally what users are supposed to use, yet some do en-US and such.